### PR TITLE
Ch16575/basic calculation functions addition

### PR DESCRIPTION
**Pull request by gardenof**
Added the ability to only add.

### CODE REVIEW

File: src/CalculatorButtonValues.elm [hunk 1]
```diff
@@ -13,13 +13,21 @@ type CalButton
...
+    | CalEqual
 
 
 type ButtonType
     = Num
     | Decimal
 
 
+type Action
+    = Blank
```
Comment (qxjit) [inline src/CalculatorButtonValues.elm:25]: `Blank` is an awfully strange `Action` to support. Are you perhaps wanting a `Maybe` somewhere?

File: src/CalculatorButtonValues.elm [hunk 2]
```diff
@@ -13,13 +13,21 @@ type CalButton
     | CalNumNine
     | CalNumZero
     | CalDecimal
+    | CalAdd
+    | CalEqual
```
Comment (qxjit) [inline src/CalculatorButtonValues.elm]: This are suspiciously similar to `Action`. Maybe you should have a `CalAction Action` constructor instead?

File: src/Main.elm [hunk 3]
```diff
@@ -28,25 +32,109 @@ init =
...
+        ( Add, Add ) ->
+            simpleMath model
+
+        ( _, _ ) ->
+            model
+
+
+simpleMath : Model -> Model
+simpleMath model =
+    case model.actionA of
```
Comment (qxjit) [inline src/Main.elm]: I would probably get rid of the `case` statement entirely here and just name this function something like `executeAdd`

File: src/Main.elm [hunk 4]
```diff
@@ -28,25 +32,109 @@ init =
...
 type Msg
     = UpdateDisplay CalButton
+    | Saveinput CalButton
 
 
-type DisplayStatus
-    = ShowingResults
-    | NotResults
+buttonToAction : CalButton -> Action
+buttonToAction button =
```
Comment (qxjit) [inline src/Main.elm:74]: Didn't I see this function above? Why is it repeated?

File: src/Main.elm [hunk 5]
```diff
@@ -28,25 +32,109 @@ init =
...
+
+
+displatToFloat : String -> Float
+displatToFloat string =
+    case String.toFloat string of
+        Just float ->
+            float
+
+        Nothing ->
+            0
```
Comment (qxjit) [inline src/Main.elm:109]: :cry: This makes me sad. Maybe the calculator should keep numbers internally to avoid having to parse? That may need to be refactored in another PR.

File: src/Main.elm [hunk 6]
```diff
@@ -28,25 +32,109 @@ init =
...
+calActionHit : Model -> CalButton -> Model
+calActionHit model button =
+    orderOfOperations <| saveAction model button
 
-type DecimalStatus
-    = YesDecimal
-    | NoDecimal
+
+saveAction : Model -> CalButton -> Model
+saveAction model button =
```
Comment (qxjit) [inline src/Main.elm:84]: The division of labor between `saveAction` and `orderOfOperations` with respect to checking the state of what's been entered so far (`valueA` vs `valueB`) feels strange to me. I think you could probably get away with having `valueB` at all if you combine the `case` logic between the two.

Comment (gardenof) [reply src/Main.elm:84]: I had this in here because I was thinking about what was coming. 
I thought I could handle all (order of operations) outcome as long as I knew 3 Values and 3 Actions (=,-,*,/) so that is why I had the 2nd value in there.
Looking at it now I do not think it was right since the commit was just to add Plus.
I will remove it and just leave what is needed for the story.

File: src/CalculatorButtonValues.elm (hunk 1 above)
Comment (qxjit) [reply src/CalculatorButtonValues.elm:25]: Did you want to address `Blank` in this PR or a separate one?
